### PR TITLE
Add SQS `Queue` bootstrappable resource

### PR DESCRIPTION
**Pull request by A-Hilaly**
Description of changes:
Add SQS `Queue` bootsrappable resource

By submitting this pull request, I confirm that my contribution is made under the terms
of the Apache 2.0 license.

### CODE REVIEW

File: src/acktest/bootstrapping/sqs.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            Attributes=create_attributes,
+        )
+        self.url = queue.url
+        self.arn = queue.attributes["QueueArn"]
+
+    def cleanup(self):
+        """Deletes an SQS queue.
+        """
+        self.sqs_client.delete_queue(
+            QueueUrl=self.queue_url,
```
Comment (vijtrip2) [inline src/acktest/bootstrapping/sqs.py:60]: should this be self.url?

File: src/acktest/bootstrapping/sqs.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+# License is located at
+#
+#	 http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+from re import S
```
Comment (vijtrip2) [inline src/acktest/bootstrapping/sqs.py:14]: what is this import?

File: src/acktest/bootstrapping/sqs.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    # Inputs
+    name_prefix: str
+    policy: str = ""
+    
+    # Outputs
+    arn: str = field(init=False)
+    
+    @property
+    def sqs_client(self):
+        return boto3.client("sqs", region_name=self.region)
```
Comment (vijtrip2) [inline src/acktest/bootstrapping/sqs.py:33]: i do not see self.region field initialized anywhere

File: src/acktest/bootstrapping/sqs.py (hunk 2 above)
Comment (A-Hilaly) [reply src/acktest/bootstrapping/sqs.py]: copy pasta-ed from another file

File: src/acktest/bootstrapping/sqs.py (hunk 3 above)
Comment (A-Hilaly) [reply src/acktest/bootstrapping/sqs.py:32]: It's inherited from `Bootstrappable` class

**Review by vijtrip2 — COMMENTED** [review]
/lgtm

**Pull request merged**